recipes/django-viewflow/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "django-viewflow" %}
+{% set version = "1.0.1" %}
+{% set sha256 = "ae1b4ad831ebda3a4ddac1fd0592b22b5d315cea5f93afd4267ee345d01414ea" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - django >=1.8
+    - django-filter >=1.0
+
+test:
+  commands:
+    - python -c "import django; from django.conf import settings; settings.configure(INSTALLED_APPS=['viewflow']); django.setup(); import viewflow; import viewflow.flow; import viewflow.flow.views; import viewflow.management; import viewflow.migrations; import viewflow.nodes; import viewflow.templatetags; import viewflow.frontend; import viewflow.frontend.templatetags"
+
+about:
+  home: http://viewflow.io/
+  license: GPL 3.0
+  license_family: GPL3
+  license_file: LICENSE
+  summary: 'Reusable workflow library for Django'
+  description: |
+    Viewflow takes the best parts from two worlds. It is based on BPMN - 
+    business process modeling and notation standard - and plays well with the 
+    django web development toolchain. BPMN provides a standard notation, 
+    readily understandable by all business stakeholders. Viewflow bridges 
+    the gap between a picture and an executable, ready to use web application.
+  doc_url: http://docs.viewflow.io/
+  dev_url: https://github.com/viewflow/viewflow/
+
+extra:
+  recipe-maintainers:
+    - cshaley
+    - kmmbvnr
```

Comment:
@kmmbvnr -- Are you ok with being added as a co-maintainer of the django viewflow recipe on conda-forge? You were added by @cshaley